<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioManagerDialog</class>
 <widget class="QDialog" name="AudioManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,90,5">
     <item>
      <widget class="QComboBox" name="cbAudio"/>
     </item>
     <item>
      <widget class="QTableWidget" name="tAudioProps">
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bDone">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bPlay">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bAdd">
         <property name="text">
          <string>Add New Audio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bDelete">
         <property name="text">
          <string>Delete Audio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
